packages/core/test/combinator/sample-test.js

```diff
@@ -30,15 +30,15 @@ describe('sample', () => {
     const s = sample(Array, makeEvents(1, n), never())
 
     return collectEvents(s, ticks(n))
-      .then(events => is(0, events.length))
+      .then(events => eq(0, events.length))
```

Comment:
OT: I've been thinking about adding map and contramap operations for assertions in `@briancavalier/assert`.  Here's a spot where it could be useful: we have an array, but we really want the the length.  A contramap op could transform the input.  Since assertions are just functions `a -> a`, contramap is just function composition "on the left":

```js
const length = array => array.length
// ...
return collectEvents(s, ticks(n))
  .then(pipe(length, eq(0)))
```

Anyway ... just thinking out loud :)